### PR TITLE
Adding meld: modeling employing limited data

### PR DESCRIPTION
**Pull request by jlmaccal**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jlmaccal** [comment]
@conda-forge-admin, please ping team

I'm new to conda-forge and could use some help.

I'm trying to setup a matrix build using `conda_build_config.yaml`, but this doesn't seem to be re-rendering any of the scripts in `.ci-support`, so the build fails.

**Comment by jlmaccal** [comment]
@conda-forge-admin, please ping team

This should be ready for review.

File: recipes/meld/build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+rm -rf build || true
+
+CMAKE_FLAGS="  -DCMAKE_INSTALL_PREFIX=${PREFIX}"
```
Comment (isuruf) [inline recipes/meld/build.sh:7]:
```suggestion
CMAKE_FLAGS="${CMAKE_ARGS}  -DCMAKE_INSTALL_PREFIX=${PREFIX}"
```

**Review by isuruf — CHANGES_REQUESTED** [review]
You need to package the text https://github.com/maccallumlab/meld/blob/3f2893e6728a8eac172de7f5ce37cf1c6d99d22a/plugin/platforms/cuda/src/CudaMeldKernelSources.cpp.in#L1-L25

**Comment by jlmaccal** [comment]
> You need to package the text https://github.com/maccallumlab/meld/blob/3f2893e6728a8eac172de7f5ce37cf1c6d99d22a/plugin/platforms/cuda/src/CudaMeldKernelSources.cpp.in#L1-L25

I have update the license upstream here:
[meld/COPYING at master · maccallumlab/meld](https://github.com/maccallumlab/meld/blob/master/COPYING)

Do I need to produce a new source tarball and update this PR?

**Comment by isuruf** [comment]
That's the best approach. You could also copy that file into the recipe directory and not release a new source tarball.

**Comment by jlmaccal** [comment]
OK, done!

**Pull request merged**